wordpress-mobile/WordPress-Android
# Threat Details: Skeleton with blank Content UiState

Parent: #13326


### Description

This PR does the following for "Threat Details"
- Creates skeleton vm, fragment, recyclerview, items builder and tests - 2a171eb, 3c0c5dc, 5ae1c65, bfbce02
- Introduces `UiState` with blank `Content` - 30c07fd
- Gets threat by `threatId` using a `UseCase` - 8c1867b, 75e89e2
- Launches `ThreatDetailsActivity` from the `ScanFragment` - c5b7794

### To test

Prerequisite: Make sure both Scan feature flag and MySiteImprovements flag are set to on in the App settings.

1. Select a site with scan capability (and having threats) on the My Site tab
2. Notice that scan menu is displayed
3. Click scan menu item
4. Notice that threats are shown
5. Click a threat item
6. Notice that blank Threat Details screen is displayed

### Merge Instructions

1. Make sure FluxC PR https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1819 is merged and tag is created in develop - Merged
2. Update FluxC hash with tag (created in FluxC develop) - Done in 4f4b947
3. Remove the "Not Ready for Merge label" - Done
4. Merge this PR

PR submission checklist:
